cluster/kubeconfig-creator.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kubeconfig-creator
+spec:
+  params:
+    - name: name
+      description: name of the cluster
+      type: string
+    - name: type
+      description: type of the resource
+      type: string
+      default: cluster
+    - name: url
+      description: address of the cluster
+      type: string
+    - name: username
+      description: username for basic authentication to the cluster
+      type: string
+    - name: password
+      description: password for basic authentication to the cluster
+      type: string
+      default: ""
+    - name: cadata
+      description: contains PEM-encoded certificate authority certificates
+      type: string
+      default: ""
+    - name: clientKeyData
+      description: contains PEM-encoded data from a client key file for TLS
+      type: string
+      default: ""
+    - name: clientCertificateData
+      description: contains PEM-encoded data from a client cert file for TLS
+      type: string
+      default: ""
+    - name: namespace
+      description: default namespace to use on unspecified requests
+      type: string
+      default: ""
+    - name: token
+      description: bearer token for authentication to the cluster
+      type: string
+      default: ""
+    - name: insecure
+      description: to indicate server should be accessed without verifying the TLS certificate
+      type: string
+      default: "true"
```

Comment:
Does this mean the Task will run insecurely by default?

If that's the case I'd much prefer that the default is to run securely.

Reply:
Yes, it was meant that. Will change the default to run securely.